src/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -768,6 +768,7 @@ export const ShippingRoute: FC<ShippingProps> = props => {
                   }}
                   onClose={() => {
                     setVerifyAddress(false)
+                    setAddressVerified(true)
```

Comment:
#minor & #non-blocking
I think these two stateful variables `verifyAddress` and `addressVerified` ([here](https://github.com/artsy/force/blob/52019141b7534d4c9fbf69c334858947be56bfc1/src/Apps/Order/Routes/Shipping/index.tsx#L110-L113)) are a little bit confusing, if only in their naming. They look like they are the inverse of each other (like they could be called `shouldVerifyAddress` and `didVerifyAddress`), and for the latter it's not clear to me if this means that we called for verification or that it was successfully verified. I wonder if we could simplify this down to one var or make the naming more clear.

Reply:
agreed, I was slightly confused about how each method functions based on naming as well.

Reply:
I like the `shouldVerifyAddress` and `didVerifyAddress` variation.